packages/widgets/src/SharingDialog/sharingConstants.js

```diff
@@ -0,0 +1,55 @@
+import i18n from '@dhis2/d2-i18n'
+
+export const ACCESS_NONE = 'ACCESS_NONE'
+export const ACCESS_VIEW_ONLY = 'ACCESS_VIEW_ONLY'
+export const ACCESS_VIEW_AND_EDIT = 'ACCESS_VIEW_AND_EDIT'
+
+const noAccess = i18n.t('No access')
+export const accessStrings = {
+    [ACCESS_NONE]: {
+        publicDescription: noAccess,
+        description: noAccess,
+        option: noAccess,
+    },
+    [ACCESS_VIEW_ONLY]: {
+        publicDescription: i18n.t('Anyone logged in can find and view'),
+        description: i18n.t('Can find and view'),
+        option: i18n.t('View only'),
+    },
+    [ACCESS_VIEW_AND_EDIT]: {
+        publicDescription: i18n.t('Anyone logged in can find, edit, and view'),
+        description: i18n.t('Can find, edit, and view'),
+        option: i18n.t('Edit and view'),
+    },
+}
```

Comment:
This needs to be changed to a function that is called from within a component, otherwise these strings will always be translated to `en`

Reply:
Fixed.